app/uk/gov/hmrc/softdrinksindustrylevystub/services/DesSubmissionService.scala

```diff
@@ -19,16 +19,18 @@ package uk.gov.hmrc.softdrinksindustrylevystub.services
 import com.google.inject.Singleton
 import uk.gov.hmrc.smartstub.Enumerable.instances.utrEnum
 import uk.gov.hmrc.smartstub._
-import uk.gov.hmrc.softdrinksindustrylevystub.models.etmp.createsub._
+import uk.gov.hmrc.softdrinksindustrylevystub.models._
+
+import scala.collection.mutable
 
 @Singleton
 class DesSubmissionService {
 
-  lazy val store  = SubscriptionGenerator.store.empty
+  lazy val store: mutable.Map[String, CreateSubscriptionRequest] = SubscriptionGenerator.store.empty
 
   def createSubscriptionResponse(data: CreateSubscriptionRequest): CreateSubscriptionResponse = {
-    store(data.customerIdentificationNumber) = data
-    SubscriptionGenerator.genCreateSubscriptionResponse.seeded(data.customerIdentificationNumber).get
+    store(data.registration.cin) = data
+    SubscriptionGenerator.genCreateSubscriptionResponse.seeded(data.registration.cin).get
   }
 
   def retrieveSubscriptionDetails(utr: String): Option[CreateSubscriptionRequest] = {
```

Comment:
Should we change the name of utr to id or something similar since this is now what the controller for this endpoint takes?

Reply:
Unfortunately DES spec refers to this as cin (customer identification number) and that's the property we're using from the json. Unless I'm not understanding what you mean.

Reply:
I think we know that call will only accept a UTR so I'll park this until I see the retrieve spec.